File: ScreambunctionBackup/src/frc/team4330/robot/commands/ZeroPhaseCommand.java
package frc.team4330.robot.commands;

import edu.wpi.first.wpilibj.command.Command;
import frc.team4330.robot.Robot;

public class ZeroPhaseCommand extends Command {

	@Override
	protected boolean isFinished() {
		return !Robot.gyro.isCalibrating();
	}

}

File: Lemur/src/main/java/frc/team4330/robot/commands/ZeroPhaseCommand.kt
package frc.team4330.robot.commands

import edu.wpi.first.wpilibj.command.Command
import frc.team4330.robot.Robot

class ZeroPhaseCommand : Command() {

    override fun isFinished(): Boolean {
        return !Robot.gyro.isCalibrating
    }

}
